src/pages/GetInvolved/Club.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { withRouter, Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+import { ReactComponent as Website } from "../../assets/website.svg";
+import { ReactComponent as Facebook } from "../../assets/facebook.svg";
+import { ReactComponent as Instagram } from "../../assets/instagram.svg";
+import { ReactComponent as Twitter } from "../../assets/twitter.svg";
+
+const Club = ({ match }) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let currentClub = match.params.clubId;
+  let clubData = clubDataAll.find((object) => object.slug === currentClub);
+
+  if (!clubData) {
+    return;
```

Comment:
this is where its breaking you need to return null
```suggestion
    return null;
```